src/ui/server/test-mocks/mock-eligibility.ts

```diff
@@ -13,6 +13,30 @@ const mockEligibility = {
   totalValueInsured: 3,
   coverPeriod: COVER_PERIOD.LESS_THAN_2_YEARS.DB_ID,
   totalContractValue: TOTAL_CONTRACT_VALUE.LESS_THAN_500K.DB_ID,
+  company: {
+    id: 'clcyyopn40148m8noyar9wxrn',
+    companyName: 'Mock company',
+    companyNumber: '0123456',
+    companyWebsite: '',
+    hasDifferentTradingName: false,
+    hasDifferentTradingAddress: false,
```

Comment:
These 2 fields aren't in eligibility?

Could we consume `mock-company` here?

Reply:
done